From aa498dc27a47a527558ff6b9ff99e21693399ecb Mon Sep 17 00:00:00 2001
From: Aaron Plattner <aplattner@nvidia.com>
Date: Wed, 14 Apr 2021 00:54:55 -0700
Subject: [PATCH] x11: Rename `atom` to `ctm_atom`

This makes it clearer exactly what atom this is referring to.

Feedback from https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1816#note_1081588

Signed-off-by: Aaron Plattner <aplattner@nvidia.com>
Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1816>
---
 src/backends/x11/meta-output-xrandr.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/backends/x11/meta-output-xrandr.c b/src/backends/x11/meta-output-xrandr.c
index 6cf82f2ecf..6fc070fcc7 100644
--- a/src/backends/x11/meta-output-xrandr.c
+++ b/src/backends/x11/meta-output-xrandr.c
@@ -204,11 +204,11 @@ meta_output_xrandr_set_ctm (MetaOutputXrandr *output_xrandr,
 {
   MetaOutput *output = META_OUTPUT (output_xrandr);
   Display *xdisplay = xdisplay_from_output (output);
-  Atom atom = XInternAtom (xdisplay, "CTM", False);
+  Atom ctm_atom = XInternAtom (xdisplay, "CTM", False);
 
   xcb_randr_change_output_property (XGetXCBConnection (xdisplay),
                                     (XID) meta_output_get_id (output),
-                                    atom, XCB_ATOM_INTEGER, 32,
+                                    ctm_atom, XCB_ATOM_INTEGER, 32,
                                     XCB_PROP_MODE_REPLACE,
                                     18, &ctm->matrix);
 }
-- 
GitLab

